stactools_planet/stactools/planet/commands.py

```diff
@@ -16,7 +16,8 @@ def convert_order(manifest,
                   copy=False,
                   description=None,
                   title=None,
-                  skip_validation=False):
+                  skip_validation=False,
+                  catalog_type=None):
```

Comment:
You could just default to `pystac.CatalogType.SELF_CONTAINED` and remove the null check below?